app/javascript/v3/helpers/BrowserHelper.js

```diff
@@ -0,0 +1,12 @@
+export const getBrowserLocale = enabledLanguages => {
+  const [language, region] = navigator.language.split('-');
+  const fullLocale = `${language}_${region || ''}`;
+  return enabledLanguages.find(
+    ({ iso_639_1_code }) =>
+      iso_639_1_code === fullLocale || iso_639_1_code === language
+  )?.iso_639_1_code;
+};
+
+export const getBrowserTimezone = () => {
+  return Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
The text this returns is not standardized with the [Timezone helpers](https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html) in Ruby.

I would suggest getting the timezone offset and convert it internally. eg: `new Date().getTimezoneOffset();`